categories/categories.yml

```diff
@@ -113,6 +113,8 @@ categories:
         title: "DNS"
       edr_xdr:
         title: "EDR/XDR"
+      cloudsecurity_cdr:
+        title: "CDR"
```

Comment:
@nimarezainia are we ok with adding this category as a subcategory of `security` category?

Reply:
lgtm